frontend/src/metabase/dashboard/selectors.ts

```diff
@@ -301,13 +303,44 @@ export const getParameterTarget = createSelector(
   },
 );
 
+export const getQuestions = (state: State) => {
+  const dashboard = getDashboard(state);
+
+  if (!dashboard) {
+    return [];
+  }
+
+  const dashcardIds = dashboard.dashcards;
+
+  const questionsById = dashcardIds.reduce<Record<DashCardId, Question>>(
+    (acc, dashcardId) => {
+      const dashcard = getDashCardById(state, dashcardId);
+
+      const question = getQuestionByDashcardMemo(state, { dashcard });
+
+      acc[dashcard.id] = question;
+
+      return acc;
+    },
+    {},
+  );
```

Comment:
I'm afraid a dashcard ID won't be enough here. Dashcards can include > 1 card (and each card can be mapped to totally different parameters), so we'd need a `card_id` as well

You can add more cards to a dashboard using the "Add series" flow ([see more in docs](https://www.metabase.com/docs/latest/dashboards/multiple-series#combining-two-saved-questions)). IIRC, we'd store them in `dashcard.series` array. Also, that's the reason why `parameter_mappings` have `card_id`

Reply:
okay, I got it now, we can have multiple cards and because of that the unique key should be dashcard and not card. 

And every dashcard has cards from series + cards from card property

Reply:
@kulyk please check 6d82acc3033eb66d172ec18332fd3a2d66e98b9d